demos/human_pose_estimation_demo/src/human_pose_estimator.cpp

```diff
@@ -7,14 +7,40 @@
 #include <vector>
 
 #include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui.hpp>
 
 #include <samples/common.hpp>
 
 #include "human_pose_estimator.hpp"
 #include "peak.hpp"
 
+#define COCO
+//#define MPI
+//#define BODY_25
+//#define FACE
+//#define HAND
```

Comment:
It's better to introduce enum instead compile time definitions. So the same code can work with different types of networks (i.e. both body pose and hand).

Reply:
I rewrote the code, now can use
./humen_estimation_demo -i [cam/image] -m [mode] -t [model_type]
to run, which mode_type including COCO/MPI/BODY_25/FACE/HAND.